sources/metricdog/src/metricdog.rs

```diff
@@ -0,0 +1,158 @@
+use crate::config::Config;
+use crate::error::{self, Result};
+use crate::service_check::ServiceCheck;
+use bottlerocket_release::BottlerocketRelease;
+use log::debug;
+use reqwest::blocking::Client;
+use snafu::ResultExt;
+use std::collections::HashMap;
+use std::str::FromStr;
+use std::time::Duration;
+use url::Url;
+
+/// The send function optionally takes a timout parameter so that we can have a short timeout for
```

Comment:
nit: typo, timeout

Reply:
Fixed in [560f336](https://github.com/bottlerocket-os/bottlerocket/compare/11b09274da45b6642d81f8164f2104b6157026e4..560f33678840b377569ea0dbb2d01633767172ae)